### PR TITLE
add notes about rustdocs to the overview

### PR DESCRIPTION
**Pull request by QuietMisdreavus**

### CODE REVIEW

File: overview.md [hunk 1]
```diff
@@ -164,10 +164,27 @@ TODO
...
+documentation as tests. Part of the normal Rust distribution.
+
+* [Source code](https://github.com/rust-lang/rust/tree/master/src/librustdoc) (part of the Rust
+  compiler source code)
+* Install using Rustup (see above).
+
+### Rustdoc 2
+
+In-progress reimplementation of rustdoc using the RLS as a backend (rather than compiler-internal
+libraries). Plans include JSON output, multiple pluggable "frontends" to render as different
```
Comment (killercup) [inline overview.md:181]: Could also link to <https://github.com/nrc/dev-tools-team/blob/master/minutes/meeting%20notes%202017-05-30.md>

**Review by killercup — APPROVED** [review]
LGTM

**Review by steveklabnik — APPROVED** [review]

File: overview.md [hunk 2]
```diff
@@ -164,10 +164,32 @@ TODO
...
 
+### Rustdoc
+
+You probably don't need to use rustdoc directly (instead going through `cargo doc`). Accepts a crate
+and generates HTML pages for its public documentation. Can also execute code blocks within the
+documentation as tests. Part of the normal Rust distribution.
+
+* [Source code](https://github.com/rust-lang/rust/tree/master/src/librustdoc) (part of the Rust
+  compiler source code)
+* Install using Rustup (see above).
```
Comment (nrc) [inline overview.md]: Could you note that it is installed by default?

File: overview.md [hunk 3]
```diff
@@ -164,10 +164,32 @@ TODO
 
 ## Documentation and code exploration
 
+### Rustdoc
+
+You probably don't need to use rustdoc directly (instead going through `cargo doc`). Accepts a crate
```
Comment (nrc) [inline overview.md]: I would start with a sentence that states what Rustdoc does - "Rustdoc generates API documentation for Rust crates," or something like that

**Review by nrc — APPROVED** [review]
Looks good, thanks!

**Comment by QuietMisdreavus** [comment]
I've gone ahead and updated the description with your suggestions. How does this look?

**Comment by nrc** [comment]
Looks good, thank you!

**Pull request merged**